zeapp/server/src/main/kotlin/de/berlindroid/zekompanion/server/Main.kt

```diff
@@ -0,0 +1,103 @@
+@file:JvmName("Main")
+
+package de.berlindroid.zekompanion.server
+
+import de.berlindroid.zekompanion.server.routers.imageBin
+import de.berlindroid.zekompanion.server.routers.imagePng
+import de.berlindroid.zekompanion.server.routers.index
+import io.ktor.serialization.kotlinx.json.*
+import io.ktor.server.application.*
+import io.ktor.server.engine.*
+import io.ktor.server.http.content.*
+import io.ktor.server.plugins.contentnegotiation.*
+import io.ktor.server.routing.*
+import io.ktor.server.tomcat.Tomcat
+import java.io.File
+import java.security.KeyStore
+
+private const val LOCAL_HTTP_PORT = 8000
+private const val LOCAL_TLS_PORT = 8443
+private const val SSL_PASSWORD_ENV = "SSL_CERTIFICATE_PASSWORD"
+private const val KEYSTORE_RESOURCE_FILE = "/tmp/keystore.jks"
+
+
+fun main(args: Array<String>) {
+    val keyPassword = try {
+        System.getenv(SSL_PASSWORD_ENV)
+    } catch (e: Exception) {
+        null
+    }
+
+    val keyStore: KeyStore? = loadKeyStore(keyPassword)
```

Comment:
Maybe fail here instead of being null-safe?

```suggestion
    val keyStore: KeyStore = require(loadKeyStore(keyPassword)) {
      "Keystore not found, I have to die"
    }
```

Reply:
Hmm, depending on the existence of the keystore would hinder local development: Not everyone has the key store file but everyone should be able to develop for localhost.